### ISSUE TITLE
OpenCV 3.4.2.17 is not found now

### ISSUE THREAD

**Issue by ShuuZhu** [cyrildiagne/screenpoint#1]
There was some problem when I install screenpoint:

```ERROR: Could not find a version that satisfies the requirement opencv-contrib-python==3.4.2.17 (from screenpoint) (from versions: 3.4.8.29, 3.4.9.31, 3.4.9.33, 4.1.2.30, 4.2.0.32, 4.2.0.34)```

``` ERROR: No matching distribution found for opencv-contrib-python==3.4.2.17 (from screenpoint)```

Thanks

**Comment by cubika** [comment]
+1, same for me.

**Comment by iddqd0** [comment]
It was broken for me on Python 3.8. Try installing Python 3.7 and redoing the virtual environment.

**Comment by cyrildiagne** [comment]
Yes unfortunately SIFT has been removed from contrib after version 3.4.2.17 because of patent issues.
Patent has expired so it's been moved back (https://github.com/opencv/opencv_contrib/pull/2449) in but still not available in v4.2 as far as I could see.

**Comment by mjsevilla** [comment]
How do I downgrade from Python 3.8 to 3.7 (I'm on Win10). I tried uninstalling 3.8 but it left the PATH variables and now when I try to use the pip command, I get this error:

`Fatal error in launcher: Unable to create process using '"c:\python38\python.exe"  "C:\Python38\Scripts\pip.exe" install screenpoint': The system cannot find the file specified.`

**Comment by cyrildiagne** [comment]
I'd recommend virtualenv
```bash
virtualenv -p python3.7 venv
source venv/bin/activate
```

**Comment by mjsevilla** [comment]
Thanks, @cyrildiagne I was able to figure it out. However, since I don't have a CUDA GPU, it seems like I won't be able to mess around with this awesome prototype you created.

**Comment by cyrildiagne** [comment]
You could technically run BASNet on your mac, without CUDA acceleration but it will take a long time to process.

**Comment by mjsevilla** [comment]
@cyrildiagne i have a powerful AMD-based PC with a 8GB GPU, but I can’t seem to install torch properly

**Comment by cyrildiagne** [comment]
AFAIK Pytorch won't be able to use use that AMD GPU since it relies on CUDA (Nvidia only) at the moment for GPU acceleration.
